### PR TITLE
FIV moment banner

### PR DESCRIPTION
**Pull request by jranks123**
## Screenshots

||Small Mobile||
|------|-----|------|
| GB | Non-GB | Thank you | 
|mobilegb|mobileus|mobilethanks|

||Mobile 375||
|------|-----|------|
| GB | Non-GB | Thank you | 
|mobile375gb|mobile375us|mobile375thanks|

||Tablet||
|------|-----|------|
| GB | Non-GB | Thank you | 
|tabletgb|tabletus|tabletthanks|

||Desktop||
|------|-----|------|
| GB | Non-GB | Thank you | 
|desktopgb|desktopus|desktopthanks|

||LeftCol||
|------|-----|------|
| GB | Non-GB | Thank you | 
|leftcolgb|leftcolus|leftcolthanks|

||Wide||
|------|-----|------|
| GB | Non-GB | Thank you | 
|widegb|wideus|widethanks|

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js [hunk 1]
```diff
@@ -15,44 +16,52 @@ export const acquisitionsBannerFivTemplate = (
...
-            <div class="fiv-banner__text">
-                HIHIHI ${params.messageText}${params.ctaText}
+       
+            <div class="fiv-banner__copy-and-ctas">
+            <div class="fiv-banner__message">
+                <div class="fiv-banner__lead-sentence">
+                   This is The Guardian’s model for open, independent journalism
+                </div>
+                <div class="fiv-banner__message-text">
+                    Our mission is to keep independent journalism accessible to everyone, regardless of where they live or what they can afford. Funding from our readers safeguards our editorial independence. It also powers our work and maintains this openness. It means more people, across the world, can access accurate information with integrity at its heart.
```
Comment (SiAdcock) [inline static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js]: Nit: add a couple of line breaks to this para?

File: static/src/stylesheets/module/site-messages/_fiv-banner.scss [hunk 2]
```diff
@@ -23,45 +150,115 @@
...
+        width: $medium-circle-height;
+        height: $medium-circle-height;
+    }
 
+    @include mq($from: wide) {
+        width: $large-circle-height;
+        height: $large-circle-height;
+    }
+    border-radius: 50%;
+    mix-blend-mode: screen;
```
Comment (SiAdcock) [inline static/src/stylesheets/module/site-messages/_fiv-banner.scss]: Ooooh, nice! How does this look on Edge / older browsers?

Comment (tomrf1) [reply static/src/stylesheets/module/site-messages/_fiv-banner.scss]: still need to test on edge, but we had issues with safari and have a solution in this PR https://github.com/guardian/frontend/pull/21112/files#diff-95cd0213733bf7acf8ab941d34c6805cR184
This is all WIP currently

**Comment by walaura** [comment]
two design nits (nothing blocking)

screenshot 2019-02-18 at 7 11 16 pm

the line-height is very inconsistent between these two paras, is this on purpose?

screenshot 2019-02-18 at 7 11 24 pm

the text on the buttons on the smaller breakpoints is way too off center and down? i'm not sure how that even happened bc on the bigger breakpoints it looks fine! are all the screenshots taken at the same time?

**Review by tomrf1 — APPROVED** [review]
🎉

**Comment by ionamckendrick** [comment]
@jranks123 We just need to update the copy on the Thank you banner please (secondary line and paragraph): please C&P from here:

Thank you for supporting The Guardian’s model for open, independent journalism
Our mission is to keep independent journalism accessible to everyone, regardless of where they live or what they can afford. Funding from readers like you safeguards our editorial independence, powers our work, and maintains this openness. It means more people, across the world, can access accurate information with integrity at its heart.

**Comment by ionamckendrick** [comment]
@walaura @tomrf1 I would like to +1 Laura – the line heights for secondary lines and the main paras should always be the same. Happier to come and tweak this with you in person!

**Comment by jranks123** [comment]
@ionamckendrick @walaura  I've updated the "Thanks" screenshots with the corrected buttons and line heights. Will do others later!
